modules/system/chat-wfrp4e.js

```diff
@@ -392,7 +392,7 @@ export default class ChatWFRP {
 
   static _onCorruptButtonClicked(event) {
     let strength = $(event.currentTarget).attr("data-strength").toLowerCase();
-    if (strength != "moderate" && strength != "minor" && strength != "major")
```

Comment:
I'm not sure this will work, these are meant to be data keys, not human readable values.